test/sharness/t0116-prometheus-data/prometheus_metrics

```diff
@@ -570,3 +570,91 @@ process_resident_memory_bytes
 process_start_time_seconds
 process_virtual_memory_bytes
 process_virtual_memory_max_bytes
+tcp_connection_duration_bucket
```

Comment:
Rebase required because of the added QUIC metrics from #8125

Reply:
Done.